src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,125 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int INITIAL_CAPACITY = 16;
```

Comment:
The name `INITIAL_CAPACITY` for the constant is not very informative. It would be better to rename it to `DEFAULT_CAPACITY` to clearly show its purpose.

Reply:
Done